terraform/groups/configurable-api-caller/main.tf

```diff
@@ -76,17 +76,17 @@ resource "aws_iam_role_policy_attachment" "param_read" {
 // Cloudwatch rule event
 // Add further resources here to add new service calls
 
-resource "aws_cloudwatch_event_rule" "call_api_caller_lambda" {
-  name                = "call_api_caller_lambda"
-  description         = "Cloudwatch event to call ${aws_lambda_function.configurable_api_lambda.function_name} lambda routinely"
+resource "aws_cloudwatch_event_rule" "protect_your_details_cleanup" {
+  name                = "protect_your_details_cleanup"
```

Comment:
We should replace the underscores with hyphens. Underscores are only best practice for the resource name itself, not the attribute. I.e. `protect_your_details_cleanup` -> `protect-your-details-cleanup`

Reply:
ok.. I've done the but I think you refer to, but that suggests many of the others need changing too?